modules/graceful-shutdown.adoc

```diff
@@ -7,6 +7,11 @@
 
 You can shut down your cluster in a graceful manner so that it can be restarted at a later date.
 
+[NOTE]
+====
+You can leave a cluster shut down for up to 1 year and expect it to restart gracefully. If the cluster remains down for 1 year, restarting it requires that you manually approve the pending `node-bootstrapper` certificate signing requests (CSRs) to recover kubelet certificates.
```

Comment:
1. `for up to 1 year and expect`: Can we provide a way to check this date?

    - For example, it looks like this, but we need to confirm from QE.
    - `$ oc -n openshift-kube-apiserver-operator get secret kube-apiserver-to-kubelet-signer -o jsonpath='{.metadata.annotations.auth\.openshift\.io/certificate-not-after}'`

2. `If the cluster remains down for 1 year`: Can we clarify whether this is "before" or "after" the expiration date confirmed above?

    - If my understanding is correct, the scenario Eng said is "the day" or "before".

3. `you manually approve:` It's a easy procedure, but it's nice to have a sample.

    - For example, it looks like this.
    - $`oc get csr -oname | xargs oc adm certificate approve`

Reply:
Thank you. With regard to #3, we presently have the steps documented on how to manually approve pending CSRs. In my original comment, I have asked the QE verify the current steps in https://deploy-preview-35099--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/graceful-cluster-restart.html apply to this scenario. Once verified, I will be sure to reference this topic in my updates.

Reply:
Hello. Thank you for your reply.

[1] This should be checked before the cluster shutdown. For example, "up to 1 year" is correct the day after OCP is installed, but I wondering if we do this after a long time (ex. a month) after installation, we may not be able to keep it cluster shutdown for a whole year. If my understanding is correct.

[3] Yes, I think this makes sense. Thank you for letting me know.

Reply:
Good evening,

[1] I believe your understanding is correct. In my next round of edits, I will be sure to clarify the point that certificates expire 1 year from the date of installation. Thank you for pointing this out.